src/core/config/Categories.json

```diff
@@ -465,6 +465,7 @@
             "CSS Minify",
             "XPath expression",
             "JPath expression",
+            "jq",
```

Comment:
I suspect this should be `Jq` or `JQ` to keep with the theme of the rest of these operations, although I realise `jq` is more natural based on the CLI usage of it.

Reply:
Renamed to "Jq"